packages/rest-crud/README.md

```diff
@@ -64,7 +64,7 @@ class defined without the need for a repository or controller class file.
 
 If you would like more flexibility, e.g. if you would only like to define a
 default `CrudRest` controller or repository, you can use the two helper methods
-(`defineCrudRestController` and `defineCrudRepositoryClass`) exposed from
+(`defineCrudRestController` and `defineEntityCrudRepositoryClass`) exposed from
 `@loopback/rest-crud`. These functions will help you create controllers and
```

Comment:
I think the docs here does not match the actual implementation below.

The implementation:

```ts
// Reexport `defineEntityCrudRepositoryClass` from `@loopback/repository` as
 // `defineCrudRepositoryClass` for backward-compatibility
 export {defineEntityCrudRepositoryClass as defineCrudRepositoryClass} from '@loopback/repository';
```

The docs say:

```markdown
(`defineCrudRestController` and `defineEntityCrudRepositoryClass`) exposed from
 `@loopback/rest-crud`
```